runatlantis.io/docs/api-endpoints.md

```diff
@@ -52,7 +53,8 @@ curl --request POST 'https://<ATLANTIS_HOST_NAME>/api/plan' \
     "Paths": [{
       "Directory": ".",
       "Workspace": "default"
-    }]
+    }],
+    "PR": 2
```

Comment:
I believe this is a respond not part of the POST request.

https://github.com/runatlantis/atlantis/pull/997/files#diff-e15ee372224542e94e53c570ddd0b42d3497afbd20014d493debee78c1d1fd91R214

Reply:
I'm seeing it hear as part of the `POST` request: https://github.com/runatlantis/atlantis/blob/main/server/controllers/api_controller.go#L39

Looks like it was added in a subsequent commit: https://github.com/runatlantis/atlantis/commit/71233ef3b6ed8081dc49dc33c45aceeceaaea753 

I've tested this parameter locally, and it works as expected.  If `PR` is not specified it defaults to `0`, which in my environment will cause an error.

Reply:
you are correct, I was looking at the old commit